### PR TITLE
Update gobrake to v5

### PR DESCRIPTION
**Pull request by chimanjain**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,33 +15,31 @@ A basic example of how to configure glog to send logged errors to Airbrake.io:
 package main
 
 import (
-	"errors"
+    "errors"
```
Comment (kyrylo) [inline README.md:18]: Please revert tab-to-space changes.

**Review by kyrylo — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (chimanjain) [reply README.md:18]: `md` Linter is throwing a warning `no-hard-tabs`. To fix those warnings we shifted to spaces from tabs

Comment (kyrylo) [reply README.md:18]: Golang doesn't use spaces to indent its code. Everyone who copies this example, will copy it with spaces.

**Comment by chimanjain** [comment]
PTAL

File: README.md [hunk 2]
```diff
@@ -18,31 +18,30 @@ import (
...
-		n.Context["environment"] = "production"
-		return n
-	})
 	glog.SetGobrakeNotifier(airbrake)
 
 	if err := doSomeWork(); err != nil {
 		glog.Errorf("doSomeWork failed: %s", err)
 	}
 }
+
```
Comment (kyrylo) [inline README.md:44]:
```suggestion
```

**Review by kyrylo — APPROVED** [review]

**Comment by chimanjain** [comment]
PTAL

**Pull request merged**